libraries/ArduinoOTA/ArduinoOTA.h

```diff
@@ -45,7 +52,12 @@ class ArduinoOTAClass
     void setPasswordHash(const char *password);
 
     //Sets if the device should be rebooted after successful update. Default true
-    void setRebootOnSuccess(bool reboot);
+    //"eraseConfig" selects to erase WiFi Settings in conjunction with a reboot
+    //after a successful update. Default OTA_ERASE_CFG_NO - Legacy behavior
+    void setRebootOnSuccess(bool reboot, ota_erase_cfg_t eraseConfig = OTA_ERASE_CFG_NO);
+
+    //Sets flag to erase WiFi Settings at reboot/reset.
+    void setEraseConfig(bool eraseConfig = true);
```

Comment:
ref. previous discussion
`setRebootOnSuccess(bool)` remains as is, `setEraseConfig(ota_erase_cfg_t)` setting erase flag

Reply:
done.